src/button-dropdown/category-elements/styles.scss

```diff
@@ -17,7 +17,8 @@
   border-inline-width: 0;
   font-weight: bold;
   display: flex;
-  justify-content: space-between;
+  align-items: center;
+  gap: awsui.$space-xxxs;
```

Comment:
The gap in non-group items is `$space-xxs`: https://github.com/cloudscape-design/components/blob/f6bc21bcb3cae1ff0f23160b4ee8462eb77ec35f/src/button-dropdown/item-element/styles.scss#L95

I would not make it smaller here, for consistency and because with bold text if feels even more condensed.